### ISSUE TITLE
Accessibility Issue - Snapchat Social Icon

### ISSUE THREAD

**Issue by masteradhoc** [WordPress/gutenberg#47392]
### Description

There is an Accessibility Issue if you add the Snapchat Icon with label from the Social Icon Block.
Color of Snapchat Icon with Label is yellow and the text white. This is quite hard to read.

By Default we should propose a better solution for the user. Options we could go for is to make the labeltext color black in this case. see
![image](https://user-images.githubusercontent.com/6242098/214434865-763818c2-e428-4047-ae18-0d9d93445528.png)

### Step-by-step reproduction instructions

1. create a sample page
2. add a social icon block
3. add snapchat icon
4. turn on labels on the parent block in the settings area
![image](https://user-images.githubusercontent.com/6242098/214434501-6e880537-dfed-42f7-bc3a-e7106455e5fc.png)
5. check the accessibility issue (see screenshots below)

### Screenshots, screen recording, code snippet

- Screenshot edit page
![image](https://user-images.githubusercontent.com/6242098/214434228-8ba660b2-5714-41ca-ba0f-508f3c26e10b.png)

- Frontend View on Sample page
![image](https://user-images.githubusercontent.com/6242098/214434301-0de98918-7dac-4695-91e8-5bfaa51a1a5e.png)

### Environment info

- newest wordpress version (currently 6.1.1)
- newest gutenberg version (currently 15.0.1)
- chromium browsers
- 2023 Default WP-Theme

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by paaljoachim** [comment]
Confirming the yellow and white which is difficult to see.

Backend:
![Screenshot 2023-01-25 at 11 19 22](https://user-images.githubusercontent.com/5323259/214538147-bb7dca92-229f-488d-b994-4de8f1884d28.jpg)

Frontend:
![Screenshot 2023-01-25 at 11 19 34](https://user-images.githubusercontent.com/5323259/214538172-e1c9cfea-44a5-441b-8f28-e3ace542e131.jpg)
